src/images/folder-disabled.svg

```diff
@@ -16,7 +16,6 @@
    sodipodi:version="0.32"
    inkscape:version="0.91 r13725"
    sodipodi:docname="folder-disabled.svg"
-   inkscape:export-filename="/home/jimmac/Desktop/horlander-style3.png"
```

Comment:
Instead of editing the icons here and adding them again in `/webapp/images/`, they seem good to be moved over, alongside removing the PNG icons.

For the record, this would address https://issues.jenkins.io/browse/JENKINS-66698

Reply:
These are different files: the ones in `webapp` are optimized using SVGO's [UI](https://jakearchibald.github.io/svgomg/). The ones in main source are mostly unchanged, I only removed metadata specific to author's computer.

Not sure what's the best long term solution here: we could put only the optimized SVGs in the repo, or hook SVGO into the build process. There was some discussion about this in Jenkins core and IIRC the preference was for the latter, but that would require too many changes for this small PR, so for now I just kept both.